codes.go

```diff
@@ -11,7 +11,10 @@ const esc = "\033["
 
 type attribute int
 
-// Foreground weight/decoration attributes.
+// Define the possible state of text inside the application, either Bold, faint, italic or underline.
```

Comment:
The comment should describe the const block, not what the code author has done. So in other words, "define" isn't needed here.

These blocks aren't related to templating, but programmatic use through `Styler`. That templating uses these underneath doesn't have to be called out here.

Reply:
I'll change that. This was one of the first comment block I did and it shows as my understanding of the code was poor.